version-control/introduction/tutorial.md

```diff
@@ -78,23 +80,25 @@ Some popular project using git:
 
 ## Terminology
 
-- repository: A repository is where code is stored
+As with any technology and related tool, there's a lot of terminollgy used. Here's some of the most common terms and their definitions:
 
-- checkout: When you retrieve code from a **repository**
+- **Repository**: A repository is where code is stored
 
-- commit: Applying any changes you have made to the **repository**
+- **Checkout**: When you retrieve code from a **repository**, to you local machine
+
+- **Commit**: Applying any changes you have made to the **repository**
```

Comment:
Nice, this makes it a little clearer to read 💃

Reply:
Just noticed a spelling error... will fix :]